src/node_api.cc

```diff
@@ -157,19 +157,19 @@ class HandleScopeWrapper {
 class EscapableHandleScopeWrapper {
  public:
   explicit EscapableHandleScopeWrapper(v8::Isolate* isolate) :
-    scope(isolate), escapeCalled(false) {}
-  bool escapeAllreadyCalled(void) {
-    return escapeCalled;
+    scope(isolate), _escapeCalled(false) {}
+  bool escapeAlreadyCalled(void) {
+    return _escapeCalled;
   }
   template <typename T>
   v8::Local<T> Escape(v8::Local<T> handle) {
-    escapeCalled = true;
+    _escapeCalled = true;
     return scope.Escape(handle);
   }
 
  private:
   v8::EscapableHandleScope scope;
-  bool escapeCalled;
+  bool _escapeCalled;
```

Comment:
just an extremely minor nit... in various places in core we use `_` as a suffix on private fields, in others we seem to use it as a prefix. It would be great to have consistency there.